.github/workflows/deploy.yml

```
@@ -18,16 +18,20 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)